src/components/notice/notice.tsx

```diff
@@ -106,15 +116,35 @@ export class Notice implements ConditionalSlotComponent {
    * Accessible name for the close button.
    *
    * @default "Close"
+   * @deprecated – translations are now built-in, if you need to override a string, please use `messageOverrides`.
    */
-  @Prop({ reflect: false }) intlClose: string = TEXT.close;
+  @Prop({ reflect: false }) intlClose: string;
 
   /** Specifies the size of the component. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
   /** Specifies the width of the component. */
   @Prop({ reflect: true }) width: Width = "auto";
 
+  /**
+   * Made into a prop for testing purposes only
```

Comment:
In the umbrella PR, let's rephrase this to `this prop is for testing purposes only`.